server/utils/courseParticipationUtils.ts

```diff
@@ -91,27 +91,9 @@ export default class CourseParticipationUtils {
     })
 
     summaryListRows.push(CourseParticipationUtils.settingSummaryListRow(courseParticipationWithName))
+    summaryListRows.push(CourseParticipationUtils.outcomeSummaryListRow(courseParticipationWithName))
 
     if (courseParticipationWithName.outcome) {
```

Comment:
Gonna change anyway, but this outer condition can probably disappear in this commit (the second)